wpilibc/src/main/native/include/frc/smartdashboard/SendableChooser.h

```diff
@@ -82,6 +82,8 @@ class SendableChooser : public SendableChooserBase {
    */
   auto GetSelected() -> decltype(_unwrap_smart_ptr(m_choices[""]));
 
+  void OnChange(std::function<void(T)>);
```

Comment:
Needs documentation.  The documentation should note that there's only a single callback (calling this function replaces any existing OnChange callback).